### PR TITLE
added 2 new params to email.rst

### PR DESCRIPTION
**Pull request by Entrepositive**
added info for *useOwnerAsMailer* and *utmTags* params to the following:
- Get email response
- Get email properties
- List email response
- Create email properties
- Edit email properties

<a href="https://gitpod.io/#https://github.com/mautic/developer-documentation-new/pull/150"></a>

### CODE REVIEW

File: docs/rest_api/emails.rst [hunk 1]
```diff
@@ -152,6 +158,9 @@ Get an individual Email by ID.
    * - ``bccAddress``
      - string
      - The ``BCC`` Email address if it's different than the one in the Mautic configuration
+   * - ``utmTags``
+     - array
+     - Array of UTM params, all of which are of type string. Options are: utmSource, utmMedium, utmCampaign, utmContent
```
Comment (RCheesley) [inline docs/rest_api/emails.rst:163]: This should be the full word:

```suggestion
     - Array of UTM parameters, all of which are of type string. Options are: utmSource, utmMedium, utmCampaign, utmContent
```

File: docs/rest_api/emails.rst [hunk 2]
```diff
@@ -430,6 +445,12 @@ Create a new Email.
    * - ``bccAddress``
      - string
      - The ``BCC`` Email address if it's different than the one in the Mautic configuration
+   * - ``useOwnerAsMailer``
+     - boolean
+     - Whether or not contact owner should be used as mailer. Defaults to TRUE
```
Comment (RCheesley) [inline docs/rest_api/emails.rst:450]: Fix the Vale warning about Contact being capitalised, and a suggestion for the tense.

```suggestion
     - Should Mautic use the Contact owner as the sender of the Email. Defaults to TRUE
```

File: docs/rest_api/emails.rst [hunk 3]
```diff
@@ -430,6 +445,12 @@ Create a new Email.
...
+   * - ``utmTags``
+     - array
+     - Array of UTM params, all of which are of type string. Options are: utmSource, utmMedium, utmCampaign, utmContent
```
Comment (RCheesley) [inline docs/rest_api/emails.rst:453]: As before, use the full word!

```suggestion
     - Array of UTM parameters, all of which are of type string. Options are: utmSource, utmMedium, utmCampaign, utmContent
```

File: docs/rest_api/emails.rst [hunk 4]
```diff
@@ -578,6 +599,12 @@ To edit a Email and create a new one if the Email isn't found:
    * - ``bccAddress``
      - string
      - The ``BCC`` Email address if it's different than the one in the Mautic configuration
+   * - ``useOwnerAsMailer``
+     - boolean
+     - Whether or not contact owner should be used as mailer
```
Comment (RCheesley) [inline docs/rest_api/emails.rst:604]:
```suggestion
     - Should Mautic use the Contact owner as the sender of the Email
```

File: docs/rest_api/emails.rst [hunk 5]
```diff
@@ -578,6 +599,12 @@ To edit a Email and create a new one if the Email isn't found:
...
+   * - ``utmTags``
+     - array
+     - Array of UTM params, all of which are of type string. Options are: utmSource, utmMedium, utmCampaign, utmContent
```
Comment (RCheesley) [inline docs/rest_api/emails.rst:607]: As above!

```suggestion
     - Array of UTM parameters, all of which are of type string. Options are: utmSource, utmMedium, utmCampaign, utmContent
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Thanks for the PR @Entrepositive 🚀 

A few suggestions based on the Vale feedback - fairly small ones, take a look and see what you think!

**Comment by RCheesley** [comment]
@all-contributors please add @Entrepositive for docs

**Comment by Entrepositive** [comment]
Uhoh, sorry for the delay @RCheesley - and thanks for covering me on that.

**Pull request merged**